app/views.py

```diff
@@ -64,9 +75,39 @@ def projectGraphView(request, projectID):
         )
 
     return render(
-        request, "graph.html", {"projectID": projectID, "projectData": data[0]}
+        request, "graph.html", {"projectID": projectID, "projectData": data[0], "TaskFeedbackForm": TaskFeedbackForm}
+    )
+
+
+@apiKeyRequired
+def createTaskFeedback(request, projectID):
+
+    data, httpsCode = ProjectsAPIView.getProjects(
+        {"id": projectID}, getAuthorizationToken(request)
+    )
+
+    if request.method == "POST":
+        form = TaskFeedbackForm(request.POST)
+
+        if form.is_valid():
```

Comment:
When the feedback is submitted, it's not making it into this if-block.

Reply:
I wonder why that is… I got my Django book yesterday and I’m reading through it, I’m not super sure how form validation works, I’ll look into it.

Reply:
Ok, I have context for this, I think I'm going to have to figure out how to pass the feedback form to the other views a little bit better.  Will advise.